x/bridge/types/msg.go

```diff
@@ -113,3 +116,70 @@ func (msg MsgConvertCoinToERC20) ValidateBasic() error {
 	// Checks for negative
 	return msg.Amount.Validate()
 }
+
+// NewMsgConvertERC20ToCoin returns a new MsgConvertERC20ToCoin
+func NewMsgConvertERC20ToCoin(
+	initiator InternalEVMAddress,
+	receiver sdk.AccAddress,
+	contractAddr InternalEVMAddress,
+	amount sdk.Int,
+) MsgConvertERC20ToCoin {
+	return MsgConvertERC20ToCoin{
+		Initiator:        initiator.String(),
+		Receiver:         receiver.String(),
+		KavaERC20Address: contractAddr.String(),
+		Amount:           amount,
+	}
+}
+
+// GetSigners returns the addresses of signers that must sign.
+func (msg MsgConvertERC20ToCoin) GetSigners() []sdk.AccAddress {
+	sender, err := sdk.AccAddressFromHex(msg.Initiator[2:])
```

Comment:
Initiator field seems to be still valid without the `0x` prefix, so this could trim off part of the actual address if we make the `MsgConvertERC20ToCoin` manually.

I think it might be better to convert the `Initiator` field to a `common.Address` then create the `AccAddress` with the address bytes to ensure we don't remove any part of the address. Something like:

```go
addr := common.HexToAddress(msg.Initiator)
sender := sdk.AccAddress(addr.Bytes())
```